pkg/client/client.go

```diff
@@ -109,8 +109,6 @@ type AppUpdateOptions struct {
 
 type LogOptions apiv1.LogOptions
 
-type ConfirmUPgrade apiv1.ConfirmUpgrade
```

Comment:
non-blocking: Weird that this exists, I guess its just some left-over dead code from the original auto-upgrade feature?

Reply:
Yeah. I had added it when I was basically copying the pattern for the feature from previous features, but TBH, I never knew what it did and the PR was so big that I didn't catch it during cleanup.